### PR TITLE
Improve article on automatic termination of bad nodes

### PR DESCRIPTION
**Pull request by marians**
Iterates on https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/ for wording, clarity and readability.

### CODE REVIEW

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Automatic termination of unhealthy nodes
-description: A general description of a feature that can detect and terminate unhealthy nodes in the tenant cluster.
-date: 2020-11-05
+description: When worker nodes get into a state where they do not respond to health checks. Here we explain how you can activate automatic termination of such nodes.
+date: 2020-11-06
```
Comment (calvix) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:4]: it works for all nodes not only workers

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 2]
```diff
@@ -11,43 +11,40 @@ categories: ["basics"]
...
 ## Overview
 
-Giant Swarm's `terminate-unhealthy-nodes` feature helps you keep the nodes in your cluster in a healthy, running state. When enabled, a service makes periodic checks on the health of each node in your cluster. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
+Degraded nodes in a Kubernetes cluster should be a rare issue, however when it occurs, it can have severe consequences for the workloads scheduled to the affected nodes. The goal should be to detect bad nodes early and remove them from the cluster, replacing them with healthy ones.
 
-This is an `alpha` feature available on these providers:
+Starting with release v12.6.0 for AWS, you now have the option to automate the detection and termination of bad nodes. When enabled, all nodes in your cluster are periodically checked. If a node fails consecutive health checks over an extended time period, it will be drained and terminated.
 
-* AWS - `12.6.0` release onwards
+This function is available on AWS only currently. Support on other providers will follow.
```
Comment (calvix) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:18]: dropping the info from the list means you really need to read the whole text and find the release somewhere in the middle of the docs, I consider it super unfriendly for a technical person who needs to quickly find that specific information.

File: src/content/basics/automatic-termination-of-bad-nodes/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Automatic termination of unhealthy nodes
-description: A general description of a feature that can detect and terminate unhealthy nodes in the tenant cluster.
-date: 2020-11-05
+description: When worker nodes get into a state where they do not respond to health checks. Here we explain how you can activate automatic termination of such nodes.
```
Comment (calvix) [inline src/content/basics/automatic-termination-of-bad-nodes/index.md:3]: this description is working for now but in the future we want to expand the portfolio, and saying that `a state where they do not respond to health checks.` will be false as we also want to terminate nodes that have filled docker or kubelet disk but they can still respond to health checks

Comment (calvix) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md:3]: Or some other cases like  corrupted networks

Comment (calvix) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md:3]: so definition `unhealthy nodes` feels more accurate description

File: src/content/basics/automatic-termination-of-bad-nodes/index.md (hunk 2 above)
Comment (calvix) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md:18]: I personally often do that I google again some article I already read to find one specific info, being it in a paragraphs makes it harder to see

File: src/content/basics/automatic-termination-of-bad-nodes/index.md (hunk 1 above)
Comment (marians) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md:4]: Updated.

File: src/content/basics/automatic-termination-of-bad-nodes/index.md (hunk 3 above)
Comment (marians) [reply src/content/basics/automatic-termination-of-bad-nodes/index.md]: Done

**Pull request merged**